scripts/compose.py

```diff
@@ -34,16 +30,8 @@
 PACKAGE_NAME = 'localmanager'
 __version__ = "4.0.0"
 
-DEFAULT_STACK_VERSION = "8.0"
```

Comment:
What's the reason for moving the version to another class?

Reply:
this constant is used by services class so it was moved to the same file.

Reply:
got it